README.md

```diff
@@ -8,6 +8,7 @@ GNU/Linux, macOS, Android, iOS, ...).
 [![Release](https://img.shields.io/github/v/tag/kiwix/libkiwix?label=release&sort=semver)](https://download.kiwix.org/release/libkiwix/)
 [![Repositories](https://img.shields.io/repology/repositories/libkiwix?label=repositories)](https://github.com/kiwix/libkiwix/wiki/Repology)
 [![Build Status](https://github.com/kiwix/libkiwix/workflows/CI/badge.svg?query=branch%3Amaster)](https://github.com/kiwix/libkiwix/actions?query=branch%3Amaster)
+[![Doc](https://readthedocs.org/projects/libkiwix/badge/?style=flat)](https://libkiwix.Readthedocs/en/latest/?badge=latest)
```

Comment:
The link (https://libkiwix.Readthedocs/en/latest/?badge=latest) is wrong (missing `.io` or `.org` TLD). I also disapprove of the capital R in the URL.